doc/help/install-config-file.md

```diff
@@ -0,0 +1,26 @@
+install-config-file(1) -- Install configuration files in Elektra
```

Comment:
Makes sense!
I was not aware that calling the script with the `kdb` prefix (=> `kdb install-config-file`) is possible.

Reply:
> I was not aware that calling the script with the kdb prefix (=> kdb install-config-file) is possible.

Yes, this could actually be more clearly described in kdb(1) and kdb-list-tools(1).